frontend/src/sass/common/uclapi.scss

```diff
@@ -0,0 +1,439 @@
+/**
+	UCL API WEBSITE STYLES 
```

Comment:
Nice work here. Much neater!

Reply:
Cheers!! Thanks so much for such a thorough review as well  💃